cudaq/buildspec.yml

```diff
@@ -0,0 +1,39 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+framework: &FRAMEWORK cudaq
+version: &VERSION 0.11.0
+short_version: &SHORT_VERSION 0.11
```

Comment:
throughout this PR - let's upgrade the version to 0.12 which was released last week - https://pypi.org/project/cudaq/